+neurostim/+stimuli/gabor.m

```diff
@@ -126,10 +127,12 @@ function beforeTrial(o)
                     o.multiGaborsPhaseOffset = zeros(1,o.multiGaborsN);    
                 end
                 if o.multiGaborsOriRand
-                    o.multiGaborsOriOffset= 180*rand(1,o.multiGaborsN); % Random orientations for n<0

```

Comment:
Good catch that these offsets are used as radians, but it would make more sense to fix the code in the vertex shader.   
The goal was to have everything specified in the m-file in degrees for consistency, then the vertex shader converts to radians, and the shader uses the radians.   (See code comments in the shaders)

That is the way it is done for the orientation, could you fix the vertex shader (and then this code so it does the same for these offsets)?)

Reply:
Hi thanks for reviewing,
Could you tell a bit more how to fix the vertex shader? Unlike orientation, multiGaborsOriOffset  is defined as an uniform in the vertex shader. So I get this error:
`ERROR: 0:60: 'assign' : l-value required "multiGaborsOriOffset" (cannot modify a uniform)`

A workaround could be converting to radian in gabor.m as:
`            glUniform1fv(glGetUniformLocation(o.shader, 'multiGaborsOriOffset'),numel(o.multiGaborsOriOffset),pi/180*o.multiGaborsOriOffset);
`
This way we can achieve "everything specified in the m-file in degrees, and the shader uses the radians". 

Also, we would need to fix multiGaborsPhaseOffset so it is radian in the vertex shader.

Reply:
Ah, I see the problem with the Uniform spec. I don't know enough about vertex shaders to know whether it is easy to pass numbers into a variable. 

Your workaround is fine with me. That way the frag shader gets the right value (i.e. in radians, which it assumes already anyway).